php/src/php/Properties.cpp

```diff
@@ -493,9 +497,17 @@ handleFreeStorage(zend_object* object)
 extern "C"
 #endif
 static zend_object*
+#if PHP_VERSION_ID >= 80000
+handleClone(zend_object* zobj)
+#else
 handleClone(zval* zv)
+#endif
 {
+#if PHP_VERSION_ID >= 80000
+    Ice::PropertiesPtr p = *Wrapper<Ice::PropertiesPtr>::fetch(zobj)->ptr;
+#else
     Ice::PropertiesPtr p = Wrapper<Ice::PropertiesPtr>::value(zv);
+#endif
```

Comment:
Maybe a bit clear if we use a single if/else block
```suggestion
#if PHP_VERSION_ID >= 80000
handleClone(zend_object* zobj)
{
    Ice::PropertiesPtr p = *Wrapper<Ice::PropertiesPtr>::fetch(zobj)->ptr;
#else
handleClone(zval* zv)
{
    Ice::PropertiesPtr p = Wrapper<Ice::PropertiesPtr>::value(zv);
#endif
```